packages/ckeditor5-remove-format/tests/manual/removeformat.html

```diff
@@ -9,9 +9,9 @@
 		.
 	</p>
 
-	<figure class="image">
+	<figure class="image image_resized" style="width:18%;">
 		<img src="assets/sample.png" alt="Autumn fields">
-		<figcaption>Autumn <em>fields</em> by Aleksander Nowodzi&#x144;ski</figcaption>
+		<figcaption>Autumn fields by Aleksander Nowodziński</figcaption>
```

Comment:
I don't know what's the reason for removing `em` formatting in this particular line. I'll revert this (but I get substitution for a regular letter - improved code readability).

Reply:
Done in 8a6cfea